.env.example

```diff
@@ -36,7 +38,7 @@ DB_PASSWORD=retroachievements
 #SESSION_DOMAIN=.retroachievements.org
 #SESSION_SECURE_COOKIE=true
 
-MAIL_DRIVER=sendmail
+MAIL_DRIVER=sendmail #{sendmail || ses || mailcatcher}
```

Comment:
Comments in .env files should be on their own line for good practice. There are other .env parsers that will take the comment as part of the value.
```
 # sendmail, ses, mailcatcher
MAIL_DRIVER=sendmail
```